docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,703 @@
+
+[id="proc-creating-kafka-topic-terraform_{context}"]
+[discrete]
+=== Creating a Kafka topic
+
+[role="_abstract"]
+Use the `rhoas_topic` resource to create a Kafka topic with default values. This example uses the `topic` identifier and creates the `my-topic` Kafka topic. You set the Kafka ID directly in the resource.
+
+.Prerequisites
+
+* You have an offline token that is accessible to Terraform.
+* You have created a Kafka instance.
+
+.Procedure
+
+. Use the following example to create the `my-topic` Kafka topic with default values.
++
+.Example `rhoas_topic` resource with default values
+[source,shell]
+----
+$ resource ‘rhoas_topic’ “topic” {
+		name = “my-topic”
+		partitions = 1
+		Kafka_id = rhoas_kafka.instance.id
+	}
+
+----
++
+. Apply the changes by entering the following command.
++
+[source,shell]
+----
+$ terraform apply
+----
++
+. Verify that the topic is listed on the *Topics* page of the {product-kafka} {service-url-kafka}[web console^].
+
+
+[id="proc-creating-acl-binding-terraform_{context}"]
+[discrete]
+=== Creating an ACL binding
+
+[role="_abstract"]
+Use the `rhoas_acl` resource to create a ACL binding. This example uses the `acl` identifier. You set the Kafka id directly in the resource along with the topic name. The following list is a description of each field in an ACL binding resource.
+
+* `resource_type`: The type of resource you want access to. This example uses *“TOPIC”*.
+* `resource_name`: the name of resource you want access to. This example uses the name that is passed when creating the topic.
+* `principal`: the user that this binding applies to. This example uses the service account client ID.
+* `operation`: The type of operation that is allowed for the given user on this resource.
+* `permission`: Whether permission is given or taken away.
+
+
+.Prerequisites
+
+* You have an offline token that is accessible to Terraform.
+* You have created a Kafka instance and a topic.
+* You have created a service account and know the client ID.
+
+.Procedure
+
+. Use the following example to create an ACL binding.
+
++
+.Example ACL binding resource
+[source,shell]
+----
+$ resource "resource_acl" "acl" {
+  kafka_id = rhoas_kafka.instance.id
+  resource_type = "TOPIC"
+  resource_name = "my-topic"
+  pattern_type = "LITERAL"
+  principal = rhoas_service_account.srvcaccnt.client_id
+  operation = "ALL"
+  permission = "ALLOW"
+}
+
+----
+. Apply the changes by entering the following command.
++
+[source,shell]
+----
+$ terraform apply
+----
+
+[id="proc-performing-all-actions_{context}"]
+[discrete]
+=== Performing all actions
+
+[role="_abstract"]
+The following example is a Terraform configuration file that puts together all the previous tasks. It provisions a Kafka instance, creates a service account and topics, and creates an ACL binding.
+
+.Example all actions Terraform configuration file
```

Comment:
I wonder about the value of including a lengthy code block like this? It is essentially a reference, but without any accompanying descriptions of the contents. Does a complete example like this live somewhere in the Terraform Registry, where we could just link to it as an additional resource?